tagging/migrations/0001_initial.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
Django 2.2 requires migrations if any of the installed apps have migrations. So since we are using the user model, which has migrations, then all models must have migrations. Ergo, all models used in tests need migrations. This one is actually handy to have (although we will continue to handle migrations for this lib on our own), but the following migrations is only needed in order to get tests passing in djagno 2.2